src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets

```diff
@@ -45,6 +45,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ImportNuGetBuildTasksPackTargetsFromSdk Condition="'$(ImportNuGetBuildTasksPackTargetsFromSdk)' == ''">false</ImportNuGetBuildTasksPackTargetsFromSdk>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>.dll; .exe; .winmd; .json; .pri; .xml; $(AllowedOutputExtensionsInPackageBuildOutputFolder)</AllowedOutputExtensionsInPackageBuildOutputFolder>
     <AllowedOutputExtensionsInSymbolsPackageBuildOutputFolder>.pdb; .mdb; $(AllowedOutputExtensionsInPackageBuildOutputFolder); $(AllowedOutputExtensionsInSymbolsPackageBuildOutputFolder)</AllowedOutputExtensionsInSymbolsPackageBuildOutputFolder>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder Condition="'$(IncludeSymbolsInPackage)' == 'true'">$(AllowedOutputExtensionsInSymbolsPackageBuildOutputFolder)</AllowedOutputExtensionsInPackageBuildOutputFolder>
```

Comment:
i'd rather have .pdb and .mdb in a property called SymbolExtensions, and that property be appended to both AllowedOutputExtensionsInPackageBuildOutputFolder (if IncludeSymbolsInPackage is set to true) and AllowedOutputExtensionsInSymbolsPackageBuildOutputFolder

Reply:
Sure! Even better!

Reply:
@rohit21agrawal Sure! Would the extensions `.dll; .exe; .winmd; .json; .pri; .xml` get the same treatment?

Reply:
no, I think it's fine the way it is. they aren't used any place else.

Reply:
Right now the symbol extensions are prepended. Did you want me to switch it to append them?

Reply:
The ordering doesn’t really matter

Reply:
Updated.